src/scripts/main.js

```diff
@@ -1,3 +1,243 @@
 'use strict';
 
-// write your code here
+let board = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+];
```

Comment:
You also can try this
```suggestion
const defaultBoard = new Array(4).fill([0, 0, 0, 0]);
let board = defaultBoard;
```

Reply:
at least you can use `new Array(4).fill([0, 0, 0, 0]);` to fill the array with zeros